cmd/globals.go

```diff
@@ -109,6 +109,9 @@ var (
 	// Time when object layer was initialized on start up.
 	globalBootTime time.Time
 
+	globalActiveCred       credential
+	globalPublicCerts      []*x509.Certificate
+	globalObjCacheDisabled bool
```

Comment:
in `xlObjects` struct, there is `objCacheEnabled bool`.  I couldn't able to get what you mean by `put it in xlObjects`?